doc/contributing.md

```diff
@@ -1,4 +1,4 @@
-# Contributors Guide
+# **Contributors Guide**
```

Comment:
![image](https://user-images.githubusercontent.com/29518865/125478201-40ee8511-8ab4-4379-98cc-a32b48cc2f5f.png)
It looks like this is causing the menu title to be bold; is that the intention?

Reply:
I bolded all the section headers in response to https://github.com/GenericMappingTools/pygmt/pull/1339#discussion_r659132348 because the fourth level looked bad without any extra formatting and it looked odd to have only that level bolded. But since this shouldn't be the only level one header bolded in the toctree, I just removed the bold formatting for the first level header in https://github.com/GenericMappingTools/pygmt/pull/1339/commits/a760520878e4e2d61aadffd395182690b772531b. Long term, the solution is probably to use ReST for the development pages on the website.